bench/wtperf/wtperf.c

```diff
@@ -2091,6 +2333,14 @@ main(int argc, char *argv[])
 	if ((ret = config_compress(cfg)) != 0)
 		goto err;
 
+	/* You can't have truncate on a random collection */
+	if (cfg->has_truncate && cfg->random_range)
+		goto err;
```

Comment:
It would be valuable to output error messages in these two cases. It's not obvious from the API.